audio/monkeys-audio/Portfile

```diff
@@ -21,3 +21,12 @@ github.tarball_from     archive
 
 # https://github.com/fernandotcl/monkeys-audio/pull/8
 patchfiles              0001-CMakeLists-use-MATCHES-for-Clang.patch
+
+platform darwin {
+    # Since we set NoMirror, need to fetch from the GH repo, and older systems will fail to do it.
+    if {${os.major} < 13} {
+        fetch.type      git
+        depends_fetch   port:git
+        git.cmd         ${prefix}/bin/git
```

Comment:
This is a creative solution!

But MacPorts base should be adding the port:git dependency and setting git.cmd to ${prefix}/bin/git automatically on systems where /usr/bin/git is too old. Is that not working here?

Reply:
You are right, it should. Let me drop these.